From 2fa8eaa979dfd1edc488d288f12736bb63960cd4 Mon Sep 17 00:00:00 2001
From: Samu Voutilainen <smar@smar.fi>
Date: Mon, 22 Apr 2013 13:05:05 -0400
Subject: [PATCH 489/625] Stop build if CONFIG_TRACEPOINTS is not set

Previously it just tries to compile with zero modules, which is
confusing if you thought you had configured everything correctly. Now it
throws an error which tells which went wrong.

Fixes #506

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Signed-off-by: Jianchuan Wang <jianchuan.wang@windriver.com>
---
 drivers/staging/lttng2/Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/staging/lttng2/Makefile b/drivers/staging/lttng2/Makefile
index 39672af..fcb14f2f 100644
--- a/drivers/staging/lttng2/Makefile
+++ b/drivers/staging/lttng2/Makefile
@@ -3,6 +3,13 @@
 #
 
 ifneq ($(KERNELRELEASE),)
+
+ifeq ($(CONFIG_TRACEPOINTS),)
+
+$(error You need to enable CONFIG_TRACEPOINTS in kernel!)
+
+endif #CONFIG_TRACEPOINTS
+
 ifneq ($(CONFIG_TRACEPOINTS),)
 
 obj-m += lttng-ring-buffer-client-discard.o
-- 
2.0.2

